### DISCUSSION TITLE
Is there going to be a version that support Windows XP?

### DISCUSSION THREAD

**Discussion by westley1027** [Open-Shell/Open-Shell-Menu#1522] [Q&A] [answered]
Im trying to make it look like a different version, and i used early versions of classic shell:
![Screenshot 2023-04-06 185002](https://user-images.githubusercontent.com/130109189/230508040-27341c01-7d67-4103-b40c-44df8caff73a.png)

but i kept getting this error:
![Screenshot 2023-04-06 185249](https://user-images.githubusercontent.com/130109189/230508118-ecacea87-59e8-403f-8636-b9afde5cca17.png)

Is there going to be any version for xp?

**Comment by ge0rdi** [comment, marked as answer]
> Is there going to be any version for xp?

No.

You should rather stay away from dead systems like XP.
